gigamon/assets/logs/gigamon.yaml

```diff
@@ -0,0 +1,101 @@
+id: gigamon
+metric_id: gigamon
+facets:
+  - groups:
+      - Web Access
+    name: URL Path
+    path: http.url_details.path
+    source: log
+  - groups:
+      - Web Access
+    name: Status Code
+    path: http.status_code
+    source: log
+  - groups:
+      - DNS
+    name: DNS ID
+    path: dns.id
+    source: log
+  - groups:
+      - DNS
+    name: Response Code
+    path: dns.flags.rcode
+    source: log
+  - facetType: list
+    groups:
+      - gigamon
+    name: Application Name
+    path: app_name
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - gigamon
+    name: dns_host
+    path: dns_host
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - gigamon
+    name: dns_name
+    path: dns_name
+    source: log
+    type: string
+pipeline:
+  type: pipeline
+  name: gigamon
+  enabled: false
+  filter:
+    query: "source:gigamon "
+  processors:
+    - type: attribute-remapper
+      name: HTTP Host
+      enabled: true
+      sources:
+        - http_host
+      sourceType: attribute
+      target: http.url_details.path
+      targetType: attribute
+      preserveSource: false
+      overrideOnConflict: false
+    - type: attribute-remapper
+      name: Http Status Code
+      enabled: true
+      sources:
+        - http_code
+      sourceType: attribute
+      target: http.status_code
+      targetType: attribute
+      preserveSource: false
+      overrideOnConflict: false
+    - type: attribute-remapper
+      name: DNS ID
+      enabled: true
+      sources:
+        - dns_transaction_id
+      sourceType: attribute
+      target: dns.id
+      targetType: attribute
+      preserveSource: false
+      overrideOnConflict: false
+    - type: attribute-remapper
```

Comment:
Can you please replace this with a date remapper since you're remapping timestamps?